tests/lcs/parser/repeat.parsertest

```diff
@@ -0,0 +1,46 @@
+%% Copyright (C) 2016 LiveCode Ltd.
+%%
+%% This file is part of LiveCode.
+%%
+%% LiveCode is free software; you can redistribute it and/or modify it under
+%% the terms of the GNU General Public License v3 as published by the Free
+%% Software Foundation.
+%%
+%% LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+%% WARRANTY; without even the implied warranty of MERCHANTABILITY or
+%% FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+%% for more details.
+%%
+%% You should have received a copy of the GNU General Public License
+%% along with LiveCode.  If not see <http://www.gnu.org/licenses/>.
+
+%% REPEAT WITH no step
+%TEST RepeatWithNoStep
+on mouseUp
+  repeat with i = 1 to 10
+  end repeat
+end mouseUp
+%EXPECT_PASS
+%SUCCESS
+%ENDTEST
+
+%% Repeat With step
+%TEST RepeatWithCorrectStep
+on mouseUp
+  repeat with i = 1 to 10 step 1
+  end repeat
+end mouseUp
+%EXPECT_PASS
+%SUCCESS
+%ENDTEST
+
+%% Repeat with step should fail
+%RepeatWithIncorrectStep
+on mouseUp
+  repeat with i = 1 to 10 %{ALEX} 10
+  end repeat
+end mouseUp
+%EXPECT PE_REPEAT_BADCOND AT ALEX
+%SUCCESS
```

Comment:
This needs to be 
```
%EXPECT PASS
%ERROR PE_REPEAT_BADCOND AT AFTER_ENDVALUE
```

(and rename your label from ALEX to AFTER_ENDVALUE, just so it is more instructive of the intended position). You can still include alex in your garbage if you want, just do it as
```
repeat with i = 1 to 10 %{AFTER_ENDVALUE} alex 10
```
:-)